src/main/scala/com/bazaarvoice/sswf/model/history/HistoryFactory.scala

```diff
@@ -163,7 +178,7 @@ object HistoryFactory {
             startedTimers.put(timerId, h)
 
             assert(scheduledStep == currentStep, s"id[$scheduledStep] != currentStep[$currentStep]")
-            Some(StepEvent[StepEnum](eventId, eventId, Left(scheduledStep), "STARTED", currentStepStart, None, Duration.ZERO))
+            Some(StepEvent[StepEnum](eventId, eventId, Left(scheduledStep), "STARTED", currentStepStart, None, Duration.ZERO, invocations))
```

Comment:
what are you counting the invocations of here? some DefinedStep or a SleepStep? I notice that there's a block resetting the current step and start time, but not invocations at line 174